gender_analysis/analysis/gender_adjective.py

```diff
@@ -367,3 +360,17 @@ def get_top_adj(full_results, num):
         top[gender] = sorted(excluded_results[gender].items(), reverse=True, key=lambda x:x[1])[0:num]
 
     return top
+
+
+def display_gender_adjectives(result_dict,num_to_return):
```

Comment:
nitpick - space after this comma. (wonder why our linter didn't find that...)

Reply:
bc we don't currently have a linter ci job for this project! @samimak37 could you take a look at https://github.com/dhmit/paris_1970/blob/master/.github/workflows/ci.yml and add a similar job to `python_linters` there to the CI for this?